CHANGELOG.md

```diff
@@ -33,6 +66,7 @@ realm.write(() => {
 * File format: generates Realms with format v23 (reads and upgrades file format v5 or later for non-synced Realm, upgrades file format v10 or later for synced Realms).
```

Comment:
It won't let me comment on the above lines, but there are some issues. 

* [ ] don't we depend on RN >= 0.71.something?
* [ ] remove the dummy line in Fixed (or actually fill it in with some of the bugs we think we fixed...)